components/layout_2020/flexbox/layout.rs

```diff
@@ -848,7 +848,15 @@ impl FlexLine<'_> {
             .map(|(((item, item_result), content_rect), margin)| {
                 let content_rect = flex_context.rect_to_flow_relative(line_size, content_rect);
                 let margin = flex_context.sides_to_flow_relative(*margin);
-                let collapsed_margin = CollapsedBlockMargins::from_margin(&margin);
+                let converted_margin = LogicalSides {
+                    block_start: margin.block_start.into(),
+                    block_end: margin.block_end.into(),
+                    inline_start: margin.inline_start.into(),
+                    inline_end: margin.inline_end.into(),
+                };
+
+                let collapsed_margin = CollapsedBlockMargins::from_margin(&converted_margin);
+
```

Comment:
```suggestion
                let margin = flex_context.sides_to_flow_relative(*margin).into();
                let collapsed_margin = CollapsedBlockMargins::from_margin(&margin);
```

Reply:
Please use what I said instead of

```rs
                let margin = flex_context.sides_to_flow_relative(*margin);
                let collapsed_margin =
                    CollapsedBlockMargins::from_margin(&LogicalSides::<Au>::from(margin.clone()));
```